frontend/src/screens/peers/Peers.tsx

```diff
@@ -118,7 +118,7 @@ function PeerTableRow(props: PeerTableRowProps) {
       <TableCell>{peer.nodeId}</TableCell>
       <TableCell>{peer.address}</TableCell>
       <TableCell>
-        {peer.isConnected && !hasOpenedChannels(peer) && (
```

Comment:
why?

Reply:
Got it. So actually that action is "Remove peer".

Reply:
Yeah, maybe that's better copy. Although there is no reason just to disconnect a peer and keep it :thinking: